### ISSUE TITLE
batteries not included

### ISSUE THREAD

**Issue by techieshark** [google/energystrategies#2]
Awesome tool!

I enjoyed playing around with this, and discovering a mix that would get to zero carbon emissions, but it relies heavily on nuclear while having a substantial excess of wind:

<img width="971" alt="energy mix with wind, solar, and nuclear" src="https://user-images.githubusercontent.com/1072292/29386556-4b43273e-8291-11e7-8379-0b2513ac4722.png">

It'd be interesting to be able to factor in [grid-scale energy storage systems](https://en.wikipedia.org/wiki/Grid_energy_storage), which might capture some of the excess wind or solar power and use that to fill in gaps; there are a number of different potential solutions ranging from pumped hydro to batteries.

**Comment by johnplatt** [comment]
The first page of the app illustrates dispatchable sources and variable renewables sources. It's quite difficult to visualize storage.

The second page _does_ include storage, that's why there's a check box for turning storage on and off. You can see [the paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3015424)  for more details.

**Comment by kchristidis** [comment]
(I think this issue can be closed?)

**Comment by techieshark** [comment]
I see the checkbox on the second page of the [tool](https://google.github.io/energystrategies/) under 'explore future assumptions', but that does not affect the visualization on the first page.

Here's a scenario that relies on wind alone:
![image](https://user-images.githubusercontent.com/1072292/32393000-1dd5a096-c095-11e7-96c0-1047bb26a789.png)

I'm imagining that to visualize storage, there would be a slider for GW of storage. Assuming you are starting at the scenario above, as you increase the storage from 0 GW to some higher value, you would shade the excess wind energy from the left to the right with a different pattern or color (let's say, just change the dots from the green color for the wind dots to a black color for the excess wind dots), and then from left to right you could fill in any gaps with the same quantity of a solid black color.  This shows the excess wind power that is charging the batteries, and the same amount of that power which is turned around to supply the demand.

**Comment by drewbryant** [comment]
Yes, you're correct that the "explore future assumptions" state does not affect the state of the first page ("understand electricity generation"). 

Thanks for the suggestion on how to visualize storage on the first page!
